distribution/docker/src/docker/Dockerfile

```diff
@@ -20,28 +20,51 @@
 */ %>
 
 <% if (docker_base == 'iron_bank') { %>
-################################################################################
-# Build stage 0 `builder`:
-# Extract Elasticsearch artifact
-################################################################################
-
 ARG BASE_REGISTRY=registry1.dso.mil
 ARG BASE_IMAGE=ironbank/redhat/ubi/ubi8
 ARG BASE_TAG=8.4
+<% } %>
+
+################################################################################
+# Build stage 0 `zlib`:
+# Compile zlib for the current architecture
```

Comment:
Are there really no precompiled binaries we can use?

Reply:
@mark-vieira Cloudflare's package repository doesn't appear to have any binaries for ARM, and the GitHub repository has no release binaries, just tags.

Reply:
Booo, of course.